### PR TITLE
Added GCP VM Generation Workflow

### PR DESCRIPTION
**Pull request by malscent**
Updated to fix broken pipe issue

Unfix the fix

fix length of random script

I'm not sure of anything anymore

Random running funky in actions

Figured it out finally. multi-line bash statement in github action

Update to use latest image versions

### CODE REVIEW

File: gcp/couchbase-vm-generation/commands.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# create instance
```
Comment (patrick-stephens) [inline gcp/couchbase-vm-generation/commands.txt:1]: Was this supposed to be Markdown? It's got a txt extension

File: gcp/couchbase-vm-generation/create.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
```
Comment (patrick-stephens) [inline gcp/couchbase-vm-generation/create.sh:2]: Probably needs a `set -eu` as well as a bit of a comment explaining what it does, parameters, etc.

**Review by patrick-stephens — APPROVED** [review]
Nothing that needs to block

File: gcp/couchbase-vm-generation/create.sh (hunk 2 above)
Comment (malscent) [reply gcp/couchbase-vm-generation/create.sh:2]: yep

**Pull request merged**